core/src/main/java/com/linecorp/armeria/internal/common/util/IpAddrUtil.java

```diff
@@ -36,5 +39,15 @@ public static String normalize(@Nullable String ipAddr) {
         return NetUtil.bytesToIpAddress(array);
     }
 
+    public static boolean isCreatedWithIpAddressOnly(InetSocketAddress socketAddress) {
+        if (socketAddress.isUnresolved()) {
+            return false;
+        }
+
+        final InetAddress inetAddress = socketAddress.getAddress();
+        // If hostname and host address are the same, it was created with an IP address
+        return socketAddress.getHostString().equals(inetAddress.getHostAddress());
```

Comment:
@ikhoon nim
I removed this code because of test fail.
```
        final InetAddress inetAddress = socketAddress.getAddress();
        if (inetAddress.isAnyLocalAddress()) {
            return false; // Wildcard address
        }
```
If IP is wildcard address, `addressResolver` start to resolve wildcard address like `0.0.0.0`, `::`.
In that case, The `addressResolver` try to resolve `0.0.0.0`  and throws `UnknownHostException`. 
The error message is `Caused by: java.net.UnknownHostException: Failed to resolve '0.0.0.0.akadns.net.' [A(1)] after 2 queries`. 

I think that we don't need to resolve either `0.0.0.0` or `::`. 
Also, I think `0.0.0.0` and `::` are fixed IP address. 

What do you think?

Reply:
I agreed. Thanks for fixing that.